docker/entrypoint.sh

```diff
@@ -4,13 +4,14 @@ set -e
 
 if [ "$1" = 'celestia' ]; then
     echo "Initializing Celestia Node with command:"
-
+    COMMAND="celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --rpc.addr="0.0.0.0" --gateway.addr="0.0.0.0""
     if [[ -n "$NODE_STORE" ]]; then
-        echo "celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}""
-        celestia "${NODE_TYPE}" init --p2p.network "${P2P_NETWORK}" --node.store "${NODE_STORE}"
+        COMMAND=${COMMAND}" --node.store "${NODE_STORE}""
+        echo $COMMAND
+        $COMMAND
```

Comment:
Nit: looks like these 2 lines can be after `fi` and `else` branch can be removed.

Reply:
true, is it ok to just merge though since it is approved and handle this nit as a good beginner issue? 
This is currently blocking some work on rollkit.